maven-extension/src/main/java/io/opentelemetry/maven/OpenTelemetrySdkService.java

```diff
@@ -95,7 +95,8 @@ public void initialize() throws InitializationException {
     if (StringUtils.isBlank(otlpEndpoint)) {
       logger.debug(
           "OpenTelemetry: No -Dotel.exporter.otlp.endpoint property or OTEL_EXPORTER_OTLP_ENDPOINT environment variable found, use a NOOP tracer");
-      GlobalOpenTelemetry.set(OpenTelemetry.noop());
+      this.openTelemetrySdk = (OpenTelemetrySdk) OpenTelemetry.noop();
```

Comment:
that cast will fail. :( `noop()` is an instance of `DefaultOpenTelemetry`, not `OpenTelemetrySdk`. 

In general, I recommend keeping one instance variable of the API (of type `OpenTelemetry`) for doing actual instrumentation, and a 2nd instance of the SDK (which will be set the API instance when initialized) which can be used for SDK-concerns, like shutdown handling, etc. Keeping the concerns separate makes it clear what's to be used for instrumentation, and what is an operational concern.

Reply:
Thanks @jkwatson, not sure what I was thinking there.

Revised PR coming shortly